### PR TITLE
Slayer overlay

### PR DESCRIPTION
**Pull request by nopothegamer**
Fix lava fishing
add bingo profile icon to pv
added an error to dwarven overlay if you have new tab off
added max ench book to neuec - dokm
removed hide incompatible enchants from /et overlay because it was never coded in
Auto turn off search mode after 2 minutes
made it so if your modlist is under 15 and you do /neustats modlist it will just show /neustats

### CODE REVIEW

**Comment by nopothegamer** [comment]
look bro i made a new branch and it still shows all the commits idk how fix

I'm p sure the first commit is "Added slayer overlay"

File: src/main/java/io/github/moulberry/notenoughupdates/miscgui/GuiCustomEnchant.java [hunk 1]
```diff
@@ -606,15 +606,15 @@ public void render(float partialTicks) {
...
-                case 1:
-                    Gui.drawRect(guiLeft + 295 + 18, guiTop + 147, guiLeft + 295 + 16 + 18, guiTop + 147 + 16, 0x80ffffff);
-                    tooltipToDisplay = createTooltip("Incompatible Enchants",
-                            NotEnoughUpdates.INSTANCE.config.enchantingSolvers.incompatibleEnchants,
-                            "Highlight", "Hide");
-                    tooltipToDisplay.add(1, EnumChatFormatting.GRAY + "How to display enchants that are");
-                    tooltipToDisplay.add(2, EnumChatFormatting.GRAY + "incompatible with your current item,");
-                    tooltipToDisplay.add(3, EnumChatFormatting.GRAY + "eg. Smite on a sword with Sharpness");
-                    break;
+                //case 1:
```
Comment (IRONM00N) [inline src/main/java/io/github/moulberry/notenoughupdates/miscgui/GuiCustomEnchant.java:609]: please use a block comment

File: src/main/java/io/github/moulberry/notenoughupdates/miscgui/GuiCustomEnchant.java [hunk 2]
```diff
@@ -1435,14 +1435,14 @@ public boolean mouseInput(int mouseX, int mouseY) {
...
                     }
-                    case 1: {
-                        int val = NotEnoughUpdates.INSTANCE.config.enchantingSolvers.incompatibleEnchants;
-                        val += direction;
-                        if (val < 0) val = 1;
-                        if (val > 1) val = 0;
-                        NotEnoughUpdates.INSTANCE.config.enchantingSolvers.incompatibleEnchants = val;
-                        break;
-                    }
+                   // case 1: {
```
Comment (IRONM00N) [inline src/main/java/io/github/moulberry/notenoughupdates/miscgui/GuiCustomEnchant.java:1438]: use block comment here too

File: src/main/java/io/github/moulberry/notenoughupdates/miscgui/GuiEnchantColour.java [hunk 3]
```diff
@@ -192,8 +196,32 @@ public void drawScreen(int mouseX, int mouseY, float partialTicks) {
...
+        if(maxedBookFound == 0){
+            try {
+                if (NotEnoughUpdates.INSTANCE.manager.jsonToStack(
+                        NotEnoughUpdates.INSTANCE.manager.getItemInformation().get("MAXED_ENCHANT_BOOK")).hasDisplayName()) {
+                    maxedBook = NotEnoughUpdates.INSTANCE.manager.jsonToStack(NotEnoughUpdates.INSTANCE.manager.getItemInformation().get("MAXED_ENCHANT_BOOK"));
+                    maxedBookFound = 1;
+                } else {
+                    maxedBookFound = 2;
+                }
+                //TODO: Actually write something to catch any errors
```
Comment (IRONM00N) [inline src/main/java/io/github/moulberry/notenoughupdates/miscgui/GuiEnchantColour.java:208]: this should be done before I merge the pr

File: src/main/java/io/github/moulberry/notenoughupdates/profileviewer/GuiProfileViewer.java [hunk 4]
```diff
@@ -200,11 +201,18 @@ public void drawScreen(int mouseX, int mouseY, float partialTicks) {
                         0, 100 / 200f, 0, 20 / 185f, GL11.GL_NEAREST);
                 Utils.drawStringCenteredScaledMaxWidth(profileId, Minecraft.getMinecraft().fontRendererObj, guiLeft + 50,
                         guiTop + sizeY + 3 + 10, true, 90, new Color(63, 224, 208, 255).getRGB());
+                //ironmoon icon
```
Comment (IRONM00N) [inline src/main/java/io/github/moulberry/notenoughupdates/profileviewer/GuiProfileViewer.java:204]: ironman*

**Review by IRONM00N — CHANGES_REQUESTED** [review]

File: src/main/java/io/github/moulberry/notenoughupdates/miscgui/GuiCustomEnchant.java (hunk 1 above)
Comment (nopothegamer) [reply src/main/java/io/github/moulberry/notenoughupdates/miscgui/GuiCustomEnchant.java:609]: I tried here and in the next spot and IntelliJ wasnt happy so I had to do it like this

Comment (nopothegamer) [reply src/main/java/io/github/moulberry/notenoughupdates/miscgui/GuiCustomEnchant.java]: or maybe im just pepega :4Shrug:

File: src/main/java/io/github/moulberry/notenoughupdates/miscgui/GuiEnchantColour.java (hunk 3 above)
Comment (nopothegamer) [reply src/main/java/io/github/moulberry/notenoughupdates/miscgui/GuiEnchantColour.java:208]: i prob should of removed that comment but I made it not show if the item isn't loaded https://canary.discord.com/channels/516977525906341928/896407218151366687/912649488886296586

File: src/main/java/io/github/moulberry/notenoughupdates/profileviewer/GuiProfileViewer.java (hunk 4 above)
Comment (nopothegamer) [reply src/main/java/io/github/moulberry/notenoughupdates/profileviewer/GuiProfileViewer.java:204]: ok 😔

**Review by DeDiamondPro — CHANGES_REQUESTED** [review]

File: src/main/java/io/github/moulberry/notenoughupdates/miscfeatures/PetInfoOverlay.java [hunk 5]
```diff
@@ -1144,6 +1147,8 @@ public void onChatReceived(ClientChatReceivedEvent event) {
                     clearPet();
                 } else if (chatMessage.toLowerCase().contains("switching to profile")) {
                     clearPet();
+                } else if (chatMessage.toLowerCase().contains("transferring you to a new island...")) {
+                    clearPet();
                 }
```
[the remark this answers is not in the document]
Comment (nopothegamer) [reply src/main/java/io/github/moulberry/notenoughupdates/miscfeatures/PetInfoOverlay.java:1152]: 4Shrug I just did what moul did

**Review by DeDiamondPro — APPROVED** [review]
Thank you

**Review by IRONM00N — APPROVED** [review]

**Pull request merged**